### ISSUE TITLE
Problem: Archivematica 1.17.0 Ubuntu packages are not available yet

### ISSUE THREAD

**Issue by jens-st** [archivematica/Issues#1722]
**Expected behaviour**
* Archivematica 1.17.0 was released 2024-12-06 (https://github.com/artefactual/archivematica/releases/tag/v1.17.0)
* Archivematica Ubuntu 1.17.0 packages available for upgrade from older versions

**Current behaviour**
* Only Archivematica 1.17.0 RC-1 & Storage Service 0.23 RC-1 packages can be found for Ubuntu (https://packages.archivematica.org/1.17.x/).
* Rocky Linux 9 packages for 1.17.0 are present.
* Q: Will Artefactual provide Ubuntu packages? 

**Steps to reproduce**
* check https://packages.archivematica.org/1.17.x/ubuntu/pool/main/a/ or
* add repos as described in 
  * https://www.archivematica.org/en/docs/archivematica-1.17/admin-manual/installation-setup/installation/install-ubuntu/#ubuntu-22-04-jammy-installation-instructions
  * https://www.archivematica.org/en/docs/archivematica-1.17/admin-manual/installation-setup/upgrading/upgrading/#upgrade-ubuntu
* `sudo apt update`
* `sudo apt list --upgradable `
```
$ sudo apt list --upgradable 
Listing... Done
archivematica-common/unknown 1:1.17.0-rc1~22.04 amd64 [upgradable from: 1:1.15.1-1~22.04]
archivematica-dashboard/unknown 1:1.17.0-rc1~22.04 amd64 [upgradable from: 1:1.15.1-1~22.04]
archivematica-mcp-client/unknown 1:1.17.0-rc1~22.04 amd64 [upgradable from: 1:1.15.1-1~22.04]
archivematica-mcp-server/unknown 1:1.17.0-rc1~22.04 amd64 [upgradable from: 1:1.15.1-1~22.04]
archivematica-storage-service/unknown 1:0.23.0-rc1~22.04 amd64 [upgradable from: 1:0.21.1-1~22.04]
archivematica/unknown 1:1.17.0-rc1~22.04 amd64 [upgradable from: 1:1.15.1-1~22.04]
```

**Your environment (version of Archivematica, operating system, other relevant details)**
* Ubuntu 22.04 Server with package installation of Archivematica 1.15.1 & Storage Service 0.21.1

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged (if applicable)
- [ ] Details about this issue have been added to the release notes (if applicable)

**Comment by replaceafill** [comment]
> Q: Will Artefactual provide Ubuntu packages?

Thanks for reporting this Jens. I just published the Ubuntu packages for the 1.17.0 release.

**Comment by jens-st** [comment]
> > Q: Will Artefactual provide Ubuntu packages?
> 
> Thanks for reporting this Jens. I just published the Ubuntu packages for the 1.17.0 release.

Thanks for the quick fix!
